lua/treesitter-context.lua

```diff
@@ -155,7 +198,18 @@ function M.enable()
 
   autocmd({ 'BufLeave', 'WinLeave' }, close)
```

Comment:
I think maybe bufleave and winleave is no longer needed, winleave can be replaced by winclosed, and bufleave's close behavior is expressed in store_contest's win_close call.